ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard.schema.yml

```
@@ -0,0 +1,96 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_postcard:
+    type: dict
+    keys:
+      disabled:
+        type: bool
+        description: Enable or disable the postcard telemetry feature
+      ingress:
+        type: dict
+        description: Configure ingress parameters
+        keys:
+          collection:
+            type: dict
+            description: Collector configuration
+            keys:
+              source:
+                type: str
+              destination:
+                type: str
+          sample:
+            type: dict
+            description: Configure sampling parameters
+            keys:
+              rate:
+                type: int
+                description: Configure sampling rate
+                convert_types:
+                - str
+                valid_values: [ 16384, 32768, 65536 ]
+              tcp_udp_checksum:
+                type: int
+                description: Configure TCP/UDP checksum or IP ID checksum
+                convert_types:
+                - str
+      profile:
+        type: dict
+        description: Configure the postcard telemetry profile
+        keys:
+          name:
+            type: str
+          ingress_sample_policy:
+            type: str
+            description: Configure ingress parameters
+      sample_policy:
+        type: dict
+        description: Configure sampling feature
+        keys:
+          name:
+            type: str
+          match_rule:
+            type: dict
+            keys:
+              rule_name:
+                type: str
+              ipversion:

```

```suggestion
              type:
```
(Inspired by model for traffic_policies - take a look for more inspiration)

---

Replaced ipversion key with type.